### PR TITLE
Michael

### PR DESCRIPTION
**Pull request by Michael5601**

### CODE REVIEW

File: Aufgabe1/gta_v1/public/index.html [hunk 1]
```diff
@@ -39,6 +39,12 @@ <h2 class="discovery__title">Discovery</h2>
 
             <!-- ############### VS1Lab 1a Form 2 Start -->
             <form class="discovery__search" id="discoveryFilterForm" action="/discovery" method="post">
+                <fieldset>
+                    <legend>Search</legend>
+                        <label id="discovery_search_label" for="discovery_search_input">Search term or keywords:</label><br>
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/index.html:44]: We are not using `<br>`-Tags to break two elements. We design it in style.css file.

```suggestion
                        <label id="discovery_search_label" for="discovery_search_input">Search term or keywords:</label>
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 2]
```diff
@@ -124,6 +124,50 @@ a {
 
 /* ### TODO ### Styles for Discovery component */
 
+
+input[type = text] {
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:128]:
```suggestion
input[type=text] {
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 3]
```diff
@@ -124,6 +124,50 @@ a {
...
+    width: 150px;
+    padding: 4px 6px;
+    margin: 10px 10px 5px;
+    border: 1px solid;
+    box-sizing: border-box;
+}
+
+fieldset {
+    border: 1px solid #000;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:137]:
```suggestion
    border: 1px solid var(--border-color);
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 4]
```diff
@@ -124,6 +124,50 @@ a {
...
+    border-color: lightgray;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:138]:
```suggestion

```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 5]
```diff
@@ -124,6 +124,50 @@ a {
...
+    margin-left: 10px;
+    font-size: small;
+    font-weight: 600;
+}
+
+input[type=submit] {
+     width: 150px;
+     padding: 4px 4px;
+     margin: 5px 10px 10px;
+     border: 1px solid;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:157]:
```suggestion
     border: 1px solid var(--border-color);
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 6]
```diff
@@ -124,6 +124,50 @@ a {
...
+     border-color: gray;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:158]:
```suggestion

```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 7]
```diff
@@ -124,6 +124,50 @@ a {
...
+     box-sizing: border-box;
+     background-color: #e6e8eb;
+     font: 1em/1.25em Arial, Helvetica, sans-serif;
+     color: gray;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:162]:
```suggestion
     color: var(--gray-color);
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 8]
```diff
@@ -124,6 +124,50 @@ a {
...
+     text-align: center;
+}
+
+input[type=text]::placeholder {
+    color: light-gray;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:167]:
```suggestion
    color: var(--background-color);
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 9]
```diff
@@ -124,6 +124,50 @@ a {
...
+
+input[type=submit] {
+     width: 150px;
+     padding: 4px 4px;
+     margin: 5px 10px 10px;
+     border: 1px solid;
+     border-color: gray;
+     box-sizing: border-box;
+     background-color: #e6e8eb;
+     font: 1em/1.25em Arial, Helvetica, sans-serif;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:161]: The font is defined in `body`.

```suggestion

```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 10]
```diff
@@ -124,6 +124,50 @@ a {
...
+}
+
+legend {
+    margin-left: 10px;
+    font-size: small;
+    font-weight: 600;
+}
+
+input[type=submit] {
+     width: 150px;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:154]: Not necessary.

```suggestion

```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 11]
```diff
@@ -124,6 +124,50 @@ a {
...
+
+fieldset {
+    border: 1px solid #000;
+    border-color: lightgray;
+    margin-top: 1.0em
+}
+
+label {
+    margin: 0px 0px 0px 10px;
+    color: gray;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:144]:
```suggestion
    color: var(--gray-color);
```

File: Aufgabe1/gta_v1/public/stylesheets/style.css [hunk 12]
```diff
@@ -124,6 +124,50 @@ a {
 
 /* ### TODO ### Styles for Discovery component */
 
+
+input[type = text] {
+    width: 150px;
```
Comment (dustinsommerfeld) [inline Aufgabe1/gta_v1/public/stylesheets/style.css:129]: Not necessary.

```suggestion

```

**Review by dustinsommerfeld — CHANGES_REQUESTED** [review]

**Review by dustinsommerfeld — CHANGES_REQUESTED** [review]

**Review by dustinsommerfeld — APPROVED** [review]

**Review by Lukas-sys — APPROVED** [review]

**Review by Cavi1012 — COMMENTED** [review]
Good job Mr. Bangas

**Review by Cavi1012 — APPROVED** [review]

**Pull request merged**